maintainer-guide.md

```diff
@@ -152,6 +152,18 @@ Security vulnerabilities, once reported and confirmed, should be treated as a ca
 Since the detail of such vulnerability cannot be disclosed until it is fixed, an issue can be created just before a PR for the fix is submitted, with minimal information (e.g. simply "Security vulnerability" as an issue with no further description).
 The complete details can be filled in just before merging and/or after the fix is deployed.
 
+### Schema migration
+Schema migration is necessary when tables/ columns have to be added 
+
+**Role: RL**
+* Consolidate all changelogs in the current release folder in `src/main/resources/db/changelog` into a single changelog to put into `src/main/resources/db/changelog`
+* Follow dev guide on targeting changelog created and run `./gradlew liquibaseTag -PliquibaseCommandValue=<release-num>` to tag changelog to current release
```

Comment:
@NicolasCwy Just want to clarify: Should RL or PM be the one running this command? As line 165 mentioned that PM also needs to run this

Reply:
Amended this. RL has to add a changeset manually to tag the changelog. PM will run one command to execute changelog on DB